src/style.css

```
@@ -1 +1,66 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  --main-color: #00acdc;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.nav_list {
+  padding: 0;
+  padding-bottom: 4px;
```

seems to be redundant
```suggestion
  padding: 0;
```